packages/plugins/documentation/server/utils/query-params.json

```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "name": "sort",
+    "in": "query",
+    "description": "Sort by attributes ascending (asc) or descending (desc)",
+    "deprecated": false,
+    "required": false,
+    "schema": {
+      "type": "string"
+    }
+  },
+  {
+    "name": "pagination[withCount]",
+    "in": "query",
+    "description": "Retun page/pageSize (default: true)",
+    "deprecated": false,
+    "required": false,
+    "schema": {
+      "type": "boolean"
+    }
+  },
+  {
+    "name": "pagination[page]",
+    "in": "query",
+    "description": "Page number (default: 0)",
+    "deprecated": false,
+    "required": false,
+    "schema": {
+      "type": "integer"
+    }
+  },
+  {
+    "name": "pagination[pageSize]",
+    "in": "query",
+    "description": "Page size (default: 25)",
+    "deprecated": false,
+    "required": false,
+    "schema": {
+      "type": "integer"
+    }
+  },
+  {
+    "name": "pagination[start]",
+    "in": "query",
+    "description": "Offset value (default: 0)",
+    "deprecated": false,
+    "required": false,
+    "schema": {
+      "type": "integer"
+    }
+  },
+  {
+    "name": "pagination[limit]",
+    "in": "query",
+    "description": "Number of entities to return (default: 25)",
+    "deprecated": false,
+    "required": false,
+    "schema": {
+      "type": "integer"
+    }
+  },
+  {
+    "name": "fields",
+    "in": "query",
+    "description": "Fields to return (ex: title,author)",
+    "deprecated": false,
+    "required": false,
+    "schema": {
+      "type": "string"
+    }
+  },
+  {
+    "name": "populate",
+    "in": "query",
+    "description": "Relations to return",
+    "deprecated": false,
+    "required": false,
+    "schema": {
+      "type": "string"
+    }
+  },
+  {
+    "name": "filters[field][operator]=value",
+    "in": "query",
+    "description": "Returns entities matching filter",
+    "deprecated": false,
+    "required": false,
+    "style": "deepObject",
+    "explode": true,
+    "schema": {
+      "type": "object",
+      "properties": {
+        "field": {
+          "type": "string"
+        },
+        "operator": {
+          "type": "string"
+        }
+      }
+    }
+  }
+]
```

Comment:
ℹ️  I'm not sure how to configure the filters.  Using the deepObject.  With params like: [postal_code][$ne]  swagger will generate: http://localhost:1337/api/addresses?filters[field]=postal_code&[operator]=$ne, but we want http://localhost:1337/api/addresses?filters[postal_code][$ne]=1111

Reply:
I removed this for now so it doesn't break the api call

Reply:
It's not possible as our filtering system doesn't fall within the OpenAPI spec. OpenAPI does not allow for user arguments before the `=` and they must all come after or we have to generate all possible combinations of filtering before the `=` which is basically impossible.